Remotion/Validation/Core/Validators/ExactLengthValidator.cs

```diff
@@ -15,18 +15,78 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
 using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
 using JetBrains.Annotations;
+using Remotion.FunctionalProgramming;
+using Remotion.Utilities;
 using Remotion.Validation.Implementation;
+using Remotion.Validation.Results;
 
 namespace Remotion.Validation.Validators
 {
-  public class ExactLengthValidator : LengthValidator
+  public class ExactLengthValidator : ILengthValidator
   {
+    public int Min { get; }
+
+    public int? Max { get; }
```

Comment:
I overloooked the complexity of ILengthValidator interface and it having both the Min and Max properties. Since we use the interfaces primiarily for certain shared functionalitiy, we could split the interface into an IMinLengthValidator and an IMaxLengthValidator interface and apply them as needed to the types. The alternative is to use explicit implementations for the interface.

Can you do a timeboxed spike (1h) to check if splitting the interfaces can be done cheaply?
I expect that LengthSystemPropertyMetaValidationRule will be the most affected type. We would split it into a MinLength and a MaxLength version.